src/h2load.cc

```diff
@@ -481,13 +529,26 @@ int Client::connection_made() {
 
   record_connect_time(&worker->stats);
 
-  auto nreq =
-      std::min(req_todo - req_started, (size_t)config.max_concurrent_streams);
+  if (!config.timing_script) {
+    auto nreq =
+        std::min(req_todo - req_started, (size_t)config.max_concurrent_streams);
 
-  for (; nreq > 0; --nreq) {
-    submit_request();
-  }
+    for (; nreq > 0; --nreq) {
+      submit_request();
+    }
+  } else {
+
+    ev_tstamp duration = config.timings[reqidx];
 
+    while (duration < 0.0001) {
+      submit_request();
+      duration = config.timings[reqidx];
+    }
+    request_timeout_watcher.data = this;
+    ev_init(&request_timeout_watcher, client_request_timeout_cb);
+    request_timeout_watcher.repeat = duration;
```

Comment:
I thin it is better to initialize request_timeout_watcher in Client constructor, which eliminates uninitialized member field warning with static analyzer, and we can unconditionally "ev_timer_stop" it in destructor.  Please use "ev_timer_init" instead of ev_init generic macro.